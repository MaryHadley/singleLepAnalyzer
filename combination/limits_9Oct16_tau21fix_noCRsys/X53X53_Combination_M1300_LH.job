universe = vanilla
Executable = X53X53_Combination_M1300_LH.sh
Should_Transfer_Files = YES
WhenToTransferOutput = ON_EXIT
Notification = Error
request_memory = 3072
Output = X53X53_Combination_M1300_LH.out
Error = X53X53_Combination_M1300_LH.err
Log = X53X53_Combination_M1300_LH.log
Queue 1