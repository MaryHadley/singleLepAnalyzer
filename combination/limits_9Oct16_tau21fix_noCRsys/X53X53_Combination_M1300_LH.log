000 (1297373.000.000) 10/10 19:27:08 Job submitted from host: <192.168.63.249:41533>
...
001 (1297373.000.000) 10/10 19:27:18 Job executing on host: <192.168.63.240:42701>
...
006 (1297373.000.000) 10/10 19:27:27 Image size of job updated: 1
	3  -  MemoryUsage of job (MB)
	2260  -  ResidentSetSize of job (KB)
...
006 (1297373.000.000) 10/10 19:32:28 Image size of job updated: 282888
	159  -  MemoryUsage of job (MB)
	161852  -  ResidentSetSize of job (KB)
...
006 (1297373.000.000) 10/10 20:17:32 Image size of job updated: 283964
	159  -  MemoryUsage of job (MB)
	162748  -  ResidentSetSize of job (KB)
...
005 (1297373.000.000) 10/10 20:22:08 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:54:27, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:54:27, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1631  -  Run Bytes Sent By Job
	323  -  Run Bytes Received By Job
	1631  -  Total Bytes Sent By Job
	323  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :     1097        1   3824812
	   Memory (MB)          :      159     3072      3072
...
