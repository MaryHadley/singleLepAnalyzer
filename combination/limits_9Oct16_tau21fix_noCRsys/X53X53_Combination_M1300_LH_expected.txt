# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.079294   0.038438    0.16288  0.054399    0.11582
